oauthenticator/gitlab.py

```diff
@@ -23,10 +24,32 @@
 
 from .oauth2 import OAuthLoginHandler, OAuthenticator
 
+GITLAB_URL = os.getenv('GITLAB_URL')
+GITLAB_HOST = os.getenv('GITLAB_HOST')
+
+if not GITLAB_URL and GITLAB_HOST:
+    warnings.warn('Use of GITLAB_HOST has been deprecated in v0.8. '
+                  'Rename GITLAB_HOST environemnt variable to GITLAB_URL.',
+                  DeprecationWarning)
+    if GITLAB_HOST.startswith('https://') or \
```

Comment:
Let's make this one line and avoid the \. It's fine if it is over 80 characters.